### PR TITLE
Blueprint: Validate Blueprint name is not empty (HMS-3801)

### PR DESCRIPTION
**Pull request by ezr-ondrej**
We are now validating Blueprint name on the backend to be non-empty. Adds same validation for frontend

Refs HMS-3801

### CODE REVIEW

**Comment by regexowl** [comment]
The code looks good to me, one very nit picky question - do we want to set the minimal length to three? As that's what the current regex allows even thought the helper text insists 1-100 characters are possible:

image

Other option would be to adjust the regex for example with a non-capturing optional group so we really do allow 1 or 2 char names.

**Comment by regexowl** [comment]
Maybe something like this? `^[a-zA-Z0-9](?:[a-zA-Z0-9-_ ]*[a-zA-Z0-9])?$`

image

Could be also probably simplified to `^[a-z0-9](?:[a-z0-9-_ ]*[a-z0-9])?$` since the test is set as case insensitive.

File: src/Components/CreateImageWizardV2/validators.ts
Comment (ezr-ondrej) [inline src/Components/CreateImageWizardV2/validators.ts:36]: @regexowl I've changed this to only require at least two characters, I've realized we do not care about cool names, as long as we can transpose those to machine friendly image name.
I've filed https://github.com/osbuild/image-builder/pull/1093 to make the transposition happen :)

Comment (regexowl) [reply src/Components/CreateImageWizardV2/validators.ts:36]: I see, nice! Should this be blocked until we merge https://github.com/osbuild/image-builder/pull/1093 then? To avoid allowing cool names that will break the image during build?

Comment (ezr-ondrej) [reply src/Components/CreateImageWizardV2/validators.ts:36]: well we allow them today, so this would be improvement to today's situation, but sure, I'm all for fixing it properly on backend first ;)

Comment (regexowl) [reply src/Components/CreateImageWizardV2/validators.ts:36]: Ah yeah, you're completely right. I got the validation mixed up, sorry about that :sweat_smile: I think we can merge then! :)

**Review by regexowl — APPROVED** [review]
Thank you!

**Pull request merged**
